operationalGuide.md

```diff
@@ -0,0 +1,174 @@
+# Operational Guide
+
+- Scope of the Card API | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Scope-of-the-Card-API)
+- Central Design Decisions | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Central-Design-Decisions)
+- Card API Level 1 | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Card-API-Level-1)
+- Card API Level 2 | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Card-API-Level-2)
+- Card API Level 3 | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Card-API-Level-3)
+- Use of this spec | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Use-of-this-spec)
+- Appendix | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Appendix)
+
+# Scope of the Card API
+
+## Card types covered with the Card API
+In scope:
+* Debit
+* Credit
+* Prepaid
+
+Ouf of scope:
+* Deposit cards or non-Scheme-based cards (e.g.loyalty cards) 
+
+The focus for creating the Card API specification was on the Mastercard and Visa schemes. However, the standard is open to other schemes as well and feedback that gives hints on necessary adaptations will be implemented.
+
+## Transaction types covered with the Card API
+* All transactions authorized or settled via a payment card (given that the card type is in scope of this API)
+* Card account-based transactions (e.g. fees, deposits, bill payments) are only available from Level 2 upwards. 
+* Only financial transactions are covered (i.e., no PIN changes, balance inquiries, or status checks).
+
+Matching transactions retrieved via Card API with debit card transactions which are accessible on the bank account via the XS2A API is not actively supported. 
+The Card API only provides card transactions and from Level 2 upwards on the card account as well. Please note that this card account is not equivalent to the bank account.
+
+## Provider and User of the API
```

Comment:
The Card API defines the data standard which can be used by providers and users.
Access and security are not part of the standard. Those will be discussed as part of the stream ca-security.

Reply:
I've discussed with Sven Biellmann. In the ca-security group is this not a topic. In other words: in our opinion, this questions has to be answered by this stream.

Reply:
@MartinRenggli: Sven Biellmann is the head of the Payment API stream. The API Security & Consent Management stream is led by Michael “Mischa” Brandmaier (e-mail: michael.brandmaier@six-group.com, GitHub user: mibrand).

Please note that unlike the Business API streams, this stream does not define standards, but provides recommendations, such as references to existing standards (e.g. FAPI) and best practices form implementations (e.g. Swiss FAPI profile).

All work results are documented in a separate wiki, which is located  at the ca-security repo (https://github.com/swissfintechinnovations/ca-security/wiki). It would therefore be a good idea to contact Mischa if you are looking for detailed information.

Reply:
From my point of view, one cannot use this API specification standalone, i.e. one need a authorization server (probably OAuth or OIDC) to implement this API. As @christiangmehling mentioned this spec defines only the data standard, but the ca-security working group proposes a standard for the authorization part.
This means before you can request data trough the card API, a user make a consent on their data and receive an auth-token. This token can be used to access data trough the card API, which means the API can only deliver data (or more precisely gets only data from the backend where the user made consent with) provided by the backend data service.